docs/binaries/changelog.md

```diff
@@ -1,5 +1,6 @@
 # nio binary changelog
 
+<hr>
```

Comment:
FYI, we can use markdown HRs instead of HTML for these:
https://daringfireball.net/projects/markdown/syntax#hr